src/Marten/StoreOptions.cs

```diff
@@ -267,11 +319,34 @@ public ITenancy Tenancy
     }
 
     private Lazy<ITenancy>? _tenancy;
+    private Func<string, NpgsqlDataSourceBuilder> _npgsqlDataSourceBuilderFactory = DefaultNpgsqlDataSourceBuilderFactory;
+    private INpgsqlDataSourceFactory _npgsqlDataSourceFactory = new DefaultNpgsqlDataSourceFactory();
+    private readonly IList<Type> _compiledQueryTypes = new List<Type>();
+    private ResiliencePipeline _resiliencePipeline;
+    private int _applyChangesLockId = 4004;
```

Comment:
**QUESTION:** Is there a specific reason for changing properties with defaults into backing one? I preferred the previous syntax, unless that's needed.

Reply:
Just to do the null check, no other reason